### PR TITLE
doc: update getting started guide and process

### PR DESCRIPTION
**Pull request by l-mb**
Some updates to the getting started guide.

I think there's more to come, but hopefully this is already helpful.

### CODE REVIEW

File: doc/from-zero-to-hacking.md [hunk 1]
```diff
@@ -1,66 +1,109 @@
...
+    coming!
+- At least 16 GB of disk space
+  - Approximately 4 GB for the directory you're building Aquarium in,
+    and another 4 GB for `/var/lib/libvirt/` (or equivalent on your
+    system). Note that this will grow if you actually store data in the
+    cluster.
+- Preferably [openSUSE Leap](https://www.opensuse.org/#Leap) or
+  [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed) as the
+  host, though we're working on extending this guide and our tooling to
+  support other distributions - patches more than welcome!
```
Comment (asettle) [inline doc/from-zero-to-hacking.md:30]: Might be worth adding note that the next system we're planning on testing on is Ubuntu, and we're open to patches?

**Review by asettle — APPROVED** [review]
From a doc POV, perfect. I'll test again soon :) I ran through Joao's previously instructions and these look a lot more verbose.

File: doc/from-zero-to-hacking.md (hunk 1 above)
Comment (l-mb) [reply doc/from-zero-to-hacking.md:30]: I didn't want to mix the document with the roadmap. But I have opened an issue for tracking: https://github.com/aquarist-labs/aquarium/issues/223

**Review by jecluis — APPROVED** [review]

**Review by Devp00l — APPROVED** [review]
Nice improvement :+1:

File: doc/from-zero-to-hacking.md [hunk 2]
```diff
@@ -1,66 +1,109 @@
 # From Zero to Hacking
 
-## Note Before
+The goal of this document is to guide you through the process of setting
+up a local, VM-based test instance via `Vagrant` on a Linux system of
```
Comment (jhesketh) [inline doc/from-zero-to-hacking.md:4]: (nit) comma here is unnecessary

**Review by jhesketh — APPROVED** [review]
lgtm. Is there a reason this hasn't merged?

**Pull request merged**